spring-reactive-modules/spring-5-reactive-client-2/src/main/java/com/baeldung/limitrequests/server/Concurrency.java

```diff
@@ -0,0 +1,34 @@
+package com.baeldung.limitrequests.server;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.IntSupplier;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Concurrency {
+
+    public static final int MAX_CONCURRENT_REQUESTS = 5;
+    private static final Logger LOGGER = LoggerFactory.getLogger(Concurrency.class);
```

Comment:
Although the modifiers of LOGGER are static final is not a constant but a final reference so [it should not be written in uppercase](https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case).